code/popup/popup.cpp

```diff
@@ -1173,3 +1174,139 @@ void popup_game_feature_not_in_demo()
 {
 	popup(PF_USE_AFFIRMATIVE_ICON|PF_BODY_BIG, 1, POPUP_OK, XSTR( "Sorry, this feature is available only in the retail version", 200));
 }
+
+// the same as popup_till_condition(), but split into separate parts such that
+// it can run multiple conditions with a single popup
+bool popup_conditional_create(int flags, ...)
+{
+	char *format, *s;
+	va_list args;
+
+	if (Popup_is_active) {
+		Int3();
```

Comment:
We should use something other than Int3() in new code.

Reply:
Made a mental note of that, then forgot about it. 🤦
Fixed now.